src/TorchSharp/NN/Recurrent/GRU.cs

```diff
@@ -25,7 +25,7 @@ internal GRU(IntPtr handle, IntPtr boxedHandle, long hiddenSize, long numLayers,
             private bool _batch_first;
 
             [DllImport("LibTorchSharp")]
-            extern static IntPtr THSNN_GRU_forward(torch.nn.Module.HType module, IntPtr input, IntPtr h_0, out IntPtr h_n);
+            extern static IntPtr THSNN_GRU_forward(torch.nn.Module<Tensor, Tensor>.HType module, IntPtr input, IntPtr h_0, out IntPtr h_n);
```

Comment:
Unrelated to this change: are there many name conflicts? Does it make sense for every TorchSharp file to have `using static torch;` and `using static torch.nn` to be concise? Maybe even as a `global using`.

Reply:
As you said, it's unrelated. Maybe something to fix in the future.